test/e2e/network_segmentation_preconfigured_layer2.go

```diff
@@ -137,6 +137,77 @@ var _ = Describe("Network Segmentation: Preconfigured Layer2 UDN", feature.Netwo
 		}),
 	)
 
+	type invalidAPITestConfig struct {
+		netConfig     *networkAttachmentConfigParams
+		expectedError interface{}
+	}
+	DescribeTable("unmasked reserved / infrastructure subnets are not allowed",
+		func(config invalidAPITestConfig) {
+			podIPs := filterCIDRs(f.ClientSet, config.netConfig.cidr)
+			if len(podIPs) == 0 {
+				Skip("IP family not supported in this environment")
+			}
```

Comment:
I don't have a better way to achieve this, and it seems this was the solution found for the "duplicate IP validation with primary UDN layer 2 pods" context. 

Far from good though.